includes/dashboard/viewUsers.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/* 
+ * this is to view users category wise
+ */
+session_start();
+
+if( !( isset( $_SESSION['Username'] ) && isset($_SESSION['Name']) && $_SESSION['Username'] === 'admin' ) )
+{
+	echo "To access this page you need to login as Admin. Please log in first.";
+	die();
+}
+
+
+require_once "../../LocalSettings.php";
+require_once "../Globals.php";
+
+
+
+
+echo "<p>To vied the users groupwise, select the group and click 'view selected group'</p><br><br>";
+
+
+$sqlConnCat = new mysqli( $eorgDBserver , $eorgDBuser , $eorgDBpasswd , $catDB );
+
+if( $sqlConnCat->connect_errno ) 
+{
+	echo "Internal Server Error, Contact Administrator.";
+	die();
+}
+
+
+$sqlConn = new mysqli( $eorgDBserver , $eorgDBuser , $eorgDBpasswd , $eorgDBname );
+
+if( $sqlConn->connect_errno ) 
+{
+	echo "Internal Server Error, Contact Administrator.";
+	die();
+}
+
+
+$qryCat = "SELECT TABLE_NAME FROM information_schema.tables WHERE table_schema = ?";
+$stmtCat = $sqlConnCat->prepare( $qryCat );
+$stmtCat->bind_param ( 's', $catDB );
+
+
+if ( ! $stmtCat->execute() )
+{
+	echo"there is a problem with database<br>";
+	die ();
+}
+
+$resCat = $stmtCat->get_result();
+
+if ( $resCat->num_rows == 0 )
+{
+	echo "Sorry, first set the broad categories<br>";
+	die ();
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space before opening parenthesis of function call prohibited

<!-- Comment by [SideCI](https://sideci.com) -->